src/pilot/templates/network-environment.yaml

```diff
@@ -105,7 +107,34 @@ parameter_defaults:
   # CHANGEME: Change the following to the VLAN ID to use on the
   # External network
   ExternalNetworkVlanID: 190
-
+  # CHANGEME: Change the following to the MTU value to use on the
+  # External network
+  ExternalNetworkMTU: 1500
+  # CHANGEME: Change the following to the MTU value to use on the
+  # internal network  
+  InternalApiMTU: 1500
+  # CHANGEME: Change the following to the MTU value to use on the
+  # Storage network
+  StorageNetworkMTU: 1500
+  # CHANGEME: Change the following to the MTU value to use on the
+  # StorageMgmtNetwork network
+  StorageMgmtNetworkMTU: 1500
+  # CHANGEME: Change the following to the MTU value to use on the
+  # TenantNetwork network
+  TenantNetworkMTU: 1500
+  # CHANGEME: Change the following to the MTU value to use on the
+  # Provisioning network
+  ProvisioningNetworkMTU: 1500
+  # CHANGEME: Change the following to the MTU value to use on the
+  # Management network
+  ManagementNetworkMTU: 1500
+  # CHANGEME: Change the following to the MTU value to use on the
+  # Default Bonds MTU
+  DefaultBondMTU: 9216
```

Comment:
Again, should this be 9000?

Reply:
On switch ports, MTU size is set to 9216. Also, all bonds on different nodes carry different VLANs. And bonds MTU should be greater than the VLANs MTU to allow overhead of different encapsulations. That is why we set it to MAX possible MTU on the switches.